pkg/render/apiserver.go

```diff
@@ -933,7 +933,7 @@ func (c *apiServerComponent) tigeraUserClusterRole() *rbacv1.ClusterRole {
 			APIGroups: []string{"lma.tigera.io"},
 			Resources: []string{"*"},
 			ResourceNames: []string{
-				"flows", "audit*", "events", "dns",
+				"flows", "audit*", "events", "dns", "kibana_login",
```

Comment:
Hey @rene-dekker, so I read the comment for this section and it says: 

```
	// If this is a managed cluster the rule to access the clusters indices in Elasticsearch need to be added to the management
	// cluster
```

It was added a while ago in this PR https://github.com/tigera/operator/pull/356/files and then you augmented it with the new ManagementCluster CR logic. 

The comment is confusing, it doesn't seem accurate with the "If this is a managed cluster". Also it talks specifically about ES indices, but with the new resource names added here for Kibana it is now more than just ES indices. 

Could you fix it?